save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -107,6 +110,76 @@ class DownloadProjectController(private val configProperties: ConfigProperties)
                 .subscribe()
         }
 
+    /**
+     * Controller to download standard test suites
+     */
+    @Suppress("TOO_LONG_FUNCTION")
+    @PostMapping("/uploadStandardTestSuite")
+    fun uploadStandardTestSuite() {
+        readStandardTestSuitesFile().forEach { (testSuiteUrl, testSuitesPath) ->
```

Comment:
```suggestion
        readStandardTestSuitesFile().forEach { (testSuiteUrl, testSuitePaths) ->
```
I  think it's more readable, when it just ends with plural 's'.